util/rbac/rbac.go

```diff
@@ -394,14 +395,28 @@ func (e *Enforcer) runInformer(ctx context.Context, onUpdated func(cm *apiv1.Con
 	log.Info("rbac configmap informer cancelled")
 }
 
+func PolicyCSV(data map[string]string) string {
+	var strBuilder strings.Builder
+	// add the main policy first
+	if p, ok := data[ConfigMapPolicyCSVKey]; ok {
+		strBuilder.WriteString(p)
+	}
+	// append additional policies at the end of the csv
+	csvRegex := regexp.MustCompile(fmt.Sprintf("^%s.+$", ConfigMapPolicyCSVKey))
```

Comment:
I would suggest to make the suffix for the entries explicit, and then use `strings.HasSuffix(s, ".csv")` instead of a regexp matcher.

So what is now  `policy.csv.overlay1` would become `policy.overlay1.csv`. At least to me, this would be more straight forward.

Reply:
That makes sense. I will make the change. To be safe I will make 2 checks: 
`strings.HasSuffix(s, ".csv") && strings.HasPrefix(s, "policy.")`
WDYT?

Reply:
Pushed the changes. Can you please take another look?